appveyor.yml

```diff
@@ -0,0 +1,27 @@
+version: '{build}-{branch}'
+skip_tags: true
+os: Windows Server 2012 R2
+environment:
+  GYP_MSVS_VERSION: 2013
+  matrix:
+  - GENERATOR: Visual Studio 12 Win64
+    DEPS_PATH: deps64
+  - GENERATOR: Visual Studio 12
+    DEPS_PATH: deps32
+install: []
+build_script:
+# See http://help.appveyor.com/discussions/problems/539-cloning-gyp-fails

```

Comment:
Not temporary In Windows libuv uses gyp as the build system to generate the VS project files. It needs to clone its git repository first.